### DISCUSSION TITLE
Is there any way to set application icon?

### DISCUSSION THREAD

**Discussion by nir34** [AvaloniaUI/Avalonia#15405] [Q&A] [answered]
I tried ApplicationIcon mark in csproj which works on windows, but on linux, it still shows the default icon
![微信图片_20240417104336](https://github.com/AvaloniaUI/Avalonia/assets/49178617/677c0364-ef8f-4d38-83f5-be5c0cffb8f6)

**Comment by maxkatz6** [comment]
In Linux this icon depends on how you bundle your application. Different packaging solutions has different approaches.

**Reply by nir34** [reply]
I'm simply publishing the project to a specified folder, can you provide a solution?

**Reply by maxkatz6** [reply, marked as answer]
Simply published apps don't have an icon.
You need to package them.

You can find general information on packaging here https://dmnfarrell.github.io/software/linux-packaging
Also we have a simple tutorial on deb packages https://docs.avaloniaui.net/docs/deployment/debian-ubuntu 
See list of third party tools in the tutorial for more automated solutions.
